ui/src/main/webapp/package.json

```diff
@@ -33,6 +33,7 @@
     "google-code-prettify": "1.0.1",
     "jquery": "2.1.4",
     "jquery-match-height": "0.7.0",
+    "ng2-file-upload": "^1.1.0",
```

Comment:
IIUC, you recommend trusting semantic versioning unless they prove otherwise. That makes sense to me. :) I didn't know if the Angular thing was an aberration or not, but I guess it was.